react/features/video-quality/components/VideoQualityDialog.web.js

```diff
@@ -115,13 +115,21 @@ class VideoQualityDialog extends Component {
     render() {
         const { _audioOnly, _p2p, t } = this.props;
         const activeSliderOption = this._mapCurrentQualityToSliderValue();
+        const showP2PWarning = _p2p && !_audioOnly;
 
         return (
             <div className = 'video-quality-dialog'>
                 <h3 className = 'video-quality-dialog-title'>
                     { t('videoStatus.callQuality') }
                 </h3>
-                { !_audioOnly && _p2p ? this._renderP2PMessage() : null }
+                <div className = { showP2PWarning ? '' : 'hide-warning' }>
+                    <InlineMessage
```

Comment:
I put the helper back just now mainly to reduce the change set. However, I moved it because I didn't find it helped for readability anymore because it's markup that will always be rendered and didn't take up that many lines. In that case it made sense to me to not break it out anymore.